src/components/quiz.tsx

```diff
@@ -1,102 +1,139 @@
+  const [selectedResponse, setSelectedResponse] = React.useState(
+    defaultSelectedResponse,
+  )
   return (
     <div className="grid grid-cols-6 gap-4 mt-8">
-      <h1 className="text-4xl leading-10 font-extrabold text-center col-start-2 col-span-4 sm:col-start-3 sm:col-span-2 mb-4">
-        {quiz.question}
+      <h1 className="text-4xl leading-10 font-extrabold text-center col-start-2 col-span-4 sm:col-start-3 sm:col-span-2">
+        {currentQuestion.value}
       </h1>
-      <Formik
-        initialValues={{[name]: ''}}
-        onSubmit={async (values) => {
-          await new Promise((r) => setTimeout(r, 500))
-          alert(JSON.stringify(values, null, 2))
+      {currentQuestion.byline && (
+        <h2 className="text-2xl text-gray-600 col-start-2 col-span-4 sm:col-start-3 sm:col-span-2 text-center">
+          ({currentQuestion.byline})
+        </h2>
+      )}
+      <form
+        className="col-start-2 col-span-4 sm:col-start-3 sm:col-span-2 mt-4"
+        onSubmit={(e) => {
+          e.preventDefault()
+          const currentResponse = find(currentQuestion.options, {
+            score: selectedResponse,
+          })
+          const eventData = {
+            selectedResponse: currentResponse?.value,
+            score: currentResponse?.score,
+            question: currentQuestion.value,
+            questionId: currentQuestion.id,
+          }
+          setSelectedResponse(defaultSelectedResponse)
+          send('CONFIRM', {data: eventData})
         }}
       >
-        {({values}) => (
-          <Form className="col-start-2 col-span-4 sm:col-start-3 sm:col-span-2">
-            <legend className="sr-only">Server size</legend>
-            <div className="space-y-4">
-              {quiz.answers.map((option) => (
-                <Option label={option.option} key={option.value}>
-                  <Field
-                    className="bg-green-600"
-                    type="radio"
-                    name={name}
-                    value={option.value}
-                    checked={values[name] === option.value}
-                  />
-                </Option>
-              ))}
-            </div>
-            <Button className="col-start-2 col-span-4 md:col-start-3 mt-10">
-              Next Question
-            </Button>
-          </Form>
-        )}
-      </Formik>
+        <legend className="sr-only">Server size</legend>
+        <div className="space-y-4">
+          {currentQuestion.options.map((option) => (
+            <Option
+              label={option.value}
+              key={option.value}
+              onClick={() => {
+                setSelectedResponse(option.score)
+              }}
+            >
+              <input
+                className="bg-green-600"
+                tabIndex={-1}
+                type="radio"
+                name={option.value}
+                value={option.score}
+                checked={selectedResponse === option.score}
+                readOnly
```

Comment:
react complained at me that i was passing a `checked` value and no `onChange` or `readOnly` prop... so I added read only.